transports/msmq/distributor/configuration.md

```diff
@@ -76,3 +76,87 @@ If self-hosting the endpoint here is the code required to enlist the endpoint wi
 snippet: ConfiguringWorker
 
 Similar to self-hosting, if running NServiceBus prior to Version 6, ensure the `app.config` of the worker contains the `MasterNodeConfig` section to point to the hostname where the distributor process is running.
+
```

Comment:
After reviewing the whole section I think it'd make sense to add a section to each point explaining how this behavior may be observed/verified. Something brief, e.g. "In the QueueExplorer (or another tool) you see... " or "message header X contains A instead of B".